sdk/mysql/Azure.ResourceManager.MySql/api/Azure.ResourceManager.MySql.netstandard2.0.cs

```diff
@@ -1335,451 +1298,489 @@ public partial class ConfigurationListResult
-        public static Azure.ResourceManager.MySql.Models.PrivateEndpointProvisioningState Ready { get { throw null; } }
-        public static Azure.ResourceManager.MySql.Models.PrivateEndpointProvisioningState Rejecting { get { throw null; } }
-        public bool Equals(Azure.ResourceManager.MySql.Models.PrivateEndpointProvisioningState other) { throw null; }
+        public MySqlPrivateEndpointProvisioningState(string value) { throw null; }
+        public static Azure.ResourceManager.MySql.Models.MySqlPrivateEndpointProvisioningState Approving { get { throw null; } }
+        public static Azure.ResourceManager.MySql.Models.MySqlPrivateEndpointProvisioningState Dropping { get { throw null; } }
+        public static Azure.ResourceManager.MySql.Models.MySqlPrivateEndpointProvisioningState Failed { get { throw null; } }
+        public static Azure.ResourceManager.MySql.Models.MySqlPrivateEndpointProvisioningState Ready { get { throw null; } }
+        public static Azure.ResourceManager.MySql.Models.MySqlPrivateEndpointProvisioningState Rejecting { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.MySql.Models.MySqlPrivateEndpointProvisioningState other) { throw null; }
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public override bool Equals(object obj) { throw null; }
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public override int GetHashCode() { throw null; }
-        public static bool operator ==(Azure.ResourceManager.MySql.Models.PrivateEndpointProvisioningState left, Azure.ResourceManager.MySql.Models.PrivateEndpointProvisioningState right) { throw null; }
-        public static implicit operator Azure.ResourceManager.MySql.Models.PrivateEndpointProvisioningState (string value) { throw null; }
-        public static bool operator !=(Azure.ResourceManager.MySql.Models.PrivateEndpointProvisioningState left, Azure.ResourceManager.MySql.Models.PrivateEndpointProvisioningState right) { throw null; }
+        public static bool operator ==(Azure.ResourceManager.MySql.Models.MySqlPrivateEndpointProvisioningState left, Azure.ResourceManager.MySql.Models.MySqlPrivateEndpointProvisioningState right) { throw null; }
+        public static implicit operator Azure.ResourceManager.MySql.Models.MySqlPrivateEndpointProvisioningState (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.MySql.Models.MySqlPrivateEndpointProvisioningState left, Azure.ResourceManager.MySql.Models.MySqlPrivateEndpointProvisioningState right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class MySqlPrivateLinkResourceProperties
+    {
+        internal MySqlPrivateLinkResourceProperties() { }
+        public string GroupId { get { throw null; } }
```

Comment:
This is a resource identifier

Reply:
They are not based on example: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/mysql/resource-manager/Microsoft.DBforMySQL/stable/2018-06-01/examples/PrivateLinkResourcesGet.json